pyface/ui/qt4/fields/time_field.py

```diff
@@ -0,0 +1,56 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+#
+# Author: Enthought, Inc.
+# Description: <Enthought pyface package component>
+
+""" The Qt-specific implementation of the time field class """
+
+
+from traits.api import provides
+
+from pyface.fields.i_time_field import ITimeField, MTimeField
+from pyface.qt.QtGui import QTimeEdit
```

Comment:
Following the convention in other parts of pyface, do you think it'll be better to move this import before the `traits.api` import?

Reply:
Oops, yes